src/package/actions/asserts/assertNumbersAreApproximatelyEql.ts

```diff
@@ -0,0 +1,24 @@
+import {LogEventType} from '../../constants/internal';
```

Comment:
`chore`: This file should be called `assertNumbersAreApproximatelyEqual`, it seems, that is, the same as the function is named?

Reply:
indeed, it'd better be renamed